config/samples/targets/mongodbtarget/200-target.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 TriggerMesh Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: targets.triggermesh.io/v1alpha1
+kind: MongoDBTarget
+metadata:
+  name: mongo-db-target
+spec:
+  # The database to use by default if a `database` parameter is missing from the event payload(s)
+  defaultDatabase: dummydb
+  # The collection to use by default if a `database` parameter is missing from the event payload(s)
+  defaultCollection: dummycol
+  # A refrence to a kubernetes secret containing a connection string I.E mongodb+srv://<user>:<password>@<database_url>/?retryWrites=true&w=majority
```

Comment:
Just want to make sure that you are aware that anyone that can reach this target can write to any database/collection that they want and the connection allows. Could be considered a security concern for some.

Reply:
or did I miss something?

Reply:
as long as the decision makers are aware that is fine with me.
just wanted to make sure you were aware.